test/chapuser_test.go

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Please add copyright for both the files.

Reply:
added copyright